make/photon/prepare/templates/docker_compose/docker-compose.yml.jinja

```diff
@@ -354,12 +354,12 @@ services:
 {% endif %}
     dns_search: .
     ports:
-      - {{http_port}}:8080
+      - {{http_ip}}:{{http_port}}:8080
```

Comment:
Could you please use one argument like `ip_port` here and explain them in comments and doc to reduce the number of config items?